File: AdManagementSystem/src/main/java/me/rainbow/service/LogService.java
package me.rainbow.service;

import me.rainbow.entity.Log;

import java.util.List;

/**
 * @author guojinpeng
 * @date 17.12.29 15:51
 */
public interface LogService {
    List<Log> getAllLogs();

    void saveLog(Log log);

    Log getLog(Integer id);
}

File: AdManagementSystem/src/main/java/me/rainbow/service/LogService.kt
package me.rainbow.service

import me.rainbow.entity.Log

/**
 *
 * @author guojinpeng
 * @date 17.12.28 15:40
 */
interface LogService {

    fun getAllLogs(): List<Log>

    fun saveLog(log: Log)

    fun getLog(id: Int): Log
}